### PR TITLE
fix var tooltip message (#1001007)

### PR DESCRIPTION
**Pull request by andrey-ilinykh**

### CODE REVIEW

**Comment by wpopielarski** [comment]
I just glimpsed at code and I think Simon will confirm, that formatting could be made better :). Actually it is really better place to control of tooltip content.

**Comment by sschaef** [comment]
add to whitelist

File: org.scala-ide.sdt.core.tests/src/org/scalaide/core/pc/DeclarationPrinterTest.scala [hunk 1]
```diff
@@ -142,5 +142,68 @@ class DeclarationPrinterTest {
...
+    runTest("""
+          |var a/**/:Int=4
+          |def f():Unit = {
+          |  println(a)
+          |}
+          |""".stripMargin,
+          "var a: Int")
+  }
+
+  private def runTest(in: String, expected: String): Unit = {
```
Comment (sschaef) [inline org.scala-ide.sdt.core.tests/src/org/scalaide/core/pc/DeclarationPrinterTest.scala]: Could you please rename this method to something like `runTestWithMarker`? This makes it easier to understand the difference between the two `runTest` methods.

**Comment by sschaef** [comment]
As already said, the formatting is a bit off. Especially the whitespace needs to be improved. Would you please run the "format selection" action on a selection of your contributed code?

**Comment by sschaef** [comment]
The commit message needs to be improved too. See [this section](https://github.com/scala-ide/scala-ide/blob/master/CONTRIBUTING.md#creating-commits-and-writing-commit-messages). The ticket needs to be mentioned as `Fix #1001007` in the body and not in the title.

**Comment by sschaef** [comment]
Superseded by #1005

**Pull request closed without merging**